mimic/test/test_nova.py

```diff
@@ -62,6 +62,7 @@ def create_server(helper, name=None, imageRef=None, flavorRef=None,
     if body is None:
         data = {
             "name": name if name is not None else 'test_server',
+            "key_name": key_name if key_name is not None else 'test_key',
```

Comment:
Can we add an assert in the tests to ensure that the key_name when passed is set and when not passed is set to 'test_key'